include/executors/executor_sycl.hpp

```diff
@@ -257,6 +257,10 @@ class Executor<SYCL> {
    */
   Executor(cl::sycl::queue q) : q_{q} {};
 
+  cl::sycl::queue sycl_queue() {
```

Comment:
The Executor is not altered since the queue is returned by value, so this method should be const